airflow/providers/amazon/aws/operators/batch.py

```diff
@@ -359,7 +359,12 @@ def monitor_job(self, context: Context):
             else:
                 self.hook.wait_for_job(self.job_id)
 
-        awslogs = self.hook.get_job_all_awslogs_info(self.job_id)
+        awslogs = []
+        try:
+            awslogs = self.hook.get_job_all_awslogs_info(self.job_id)
+        except AirflowException as ae:
+            self.log.warning("cannot determine where to find aws logs for this batch job: %s", ae)
```

Comment:
```suggestion
            self.log.warning("Cannot determine where to find the AWS logs for this Batch job: %s", ae)
```

Also are we sure all cases of AirflowException are safe to catch and warn on here? Or should we check the message to ensure it's the specific case you're describing in the log message?

Reply:
the purpose of this function we're calling is to get the info about where the logs went. So this message should cover _any_ failure in this method.

Reply:
I think this is happen because we lack of support AWS Batch on EKS in the BatchOperator.
So it might work on simple case and fail on logs but it general it could fail in other places too.